client/common/wrappers.tsx

```diff
@@ -417,10 +418,24 @@ export const CustomSetHead: React.FC<{ customSet?: CustomSet | null }> = ({
 export const DamageTypeToggle: React.FC<{
   readonly showRanged: boolean;
   readonly setShowRanged: React.Dispatch<React.SetStateAction<boolean>>;
-}> = ({ showRanged, setShowRanged }) => {
+  readonly rangedOnly: boolean;
+  readonly meleeOnly: boolean;
+}> = ({ showRanged, setShowRanged, rangedOnly, meleeOnly }) => {
   const { t } = useTranslation('weapon_spell_effect');
   const theme = useTheme();
 
+  const notPossibleWarning = (
+    <span css={{ marginLeft: 8}}>
+        <Tooltip
+          css={{ textAlign: 'center' }}
+          title={meleeOnly ? t('RANGED_NOT_POSSIBLE') : t('MELEE_NOT_POSSIBLE')}
+          arrowPointAtCenter={true}
+        >
+          <FontAwesomeIcon icon={faExclamationCircle} />
```

Comment:
Do you think we should consider using the same yellow triangle as we do for warnings for when item conditions are not met?

Reply:
Yes, I like how this fits with the rest of the ui. Here is what it looks like with this change:

desktop:
![image](https://user-images.githubusercontent.com/31999176/180648179-217a9adb-9e1e-4b11-aa1c-0cd6901040f2.png)

mobile:
![image](https://user-images.githubusercontent.com/31999176/180648217-a88de807-75a9-4fb6-98ce-f4e0ce2b310d.png)